i8_terminal/commands/metrics/metrics_historical.py

```diff
@@ -239,13 +243,15 @@ def historical(
     columns_justify: Dict[str, Any] = {}
     for metric_display_name, metric_df in df.groupby("Metric"):
         columns_justify[metric_display_name] = "left" if metric_df["display_format"].values[0] == "str" else "right"
+    df = df.groupby(["Ticker", "Period"]).head(1)
```

Comment:
Why `head(1)` does it return the entire group or only the first row of the group?